bigquery/table.go

```diff
@@ -317,19 +317,29 @@ type MaterializedViewDefinition struct {
 	// RefreshInterval defines the maximum frequency, in millisecond precision,
 	// at which this this materialized view will be refreshed.
 	RefreshInterval time.Duration
+
+	// AllowNonIncrementalDefinition for materialized view definition.
+	// The default value is false.
+	AllowNonIncrementalDefinition bool
+
+	// MaxStaleness of data that could be returned when materialized view
+	// is queried (formatted as Google SQL Interval type).
+	MaxStaleness string
```

Comment:
Should we just use https://pkg.go.dev/cloud.google.com/go/bigquery#IntervalValue here and convert?